api/server.js

```diff
@@ -22,19 +27,27 @@ const moesifMiddleware = moesifExpress({
   logBody: true
 })
 
-//Server uses middleware to add functionality
-server.use(moesifMiddleware)
 
+// TODO: POSSIBL CODE CLEANUP WITH BELOW IMPORTS, AS THEIR NOT BEING USED.
```

Comment:
if we're not using it, delete it. Or, as I believe the kids are calling it these days: "yeet"